packages/react-storage/src/components/StorageBrowser/createStorageBrowser/createStorageBrowser.tsx

```diff
@@ -44,10 +43,8 @@ const UA_CONFIG = {
  */
 export default function createStorageBrowser<
   Input extends CreateStorageBrowserInput,
-  RInput extends Input['actions'] extends ExtendedActionConfigs
-    ? Input['actions']
-    : ExtendedActionConfigs,
->(input: Input): CreateStorageBrowserOutput<RInput> {
+  Actions extends NonNullable<Input['actions']>,
```

Comment:
Prefer `TActions`, would be great if we can update the naming of the `Input` generic to `TInput` to align it as well